plugins/postgres/dbt/adapters/postgres/impl.py

```diff
@@ -85,6 +86,18 @@ def verify_database(self, database):
     def parse_index(self, raw_index: Any) -> Optional[PostgresIndexConfig]:
         return PostgresIndexConfig.parse(raw_index)
 
+    @available
+    def standardize_grants_dict(self, grants_table: agate.Table) -> dict:
+        grants_dict = {}
+        for row in grants_table:
+            grantee = row["grantee"]
+            privilege = row["privilege"]
```

Comment:
This is currently raising a `KeyError`:
```
11:53:03  Unhandled error while executing model.testy.incremental
'privilege'
```
The reason? This column is called `privilege_type` in your `postgres__get_show_grant_sql` query below. You can either rename it to use `privilege_type` here, or add a column alias in the query below (`privilege_type as privilege`).

Reply:
[standardize_grants_dict](https://github.com/dbt-labs/dbt-core/pull/5369/files#diff-4c547e7dad44fb23a74855d32871c8d60d47382ceba0a5bf64273967256bdd08R91-R100) changes made for **postgres**